### PR TITLE
Remove hardocded pagerduty APIURL

### LINKED ISSUE #82741: Allow user to choose between Pagerduty global and Pagerduty EU endpoint
### Discussed in https://github.com/grafana/grafana/discussions/67187

<div type='discussions-op-text'>

<sup>Originally posted by **guusbeckett** April 25, 2023</sup>
At our company we use the PagerDuty EU endpoint, but it is not yet possible to use the EU endpoint of PagerDuty. 
As PagerDuty and PagerDuty EU are separate platforms, we cannot simply connect to the global endpoint and expect it to work.

I currently have a custom build that just changes the PagerDuty url to the EU one;
`https://events.pagerduty.com/v2/enqueue` becomes `https://events.eu.pagerduty.com/v2/enqueue`

Is it an idea to provide users with a new alert manager recipient and add _PagerDuty EU_ or would you prefer something else?

I'd be happy to implement this in the code.</div>

### PR DESCRIPTION
**Pull request by gaurav1999**
- Fixes: https://github.com/grafana/grafana/issues/82741
- Linked PR: https://github.com/grafana/grafana/pull/88007

Mentioned by @yuri-tceretian here on the PR: https://github.com/grafana/grafana/pull/88007#issuecomment-2120646920

Thanks!  🚀

### CODE REVIEW

File: internal/resources/grafana/resource_alerting_contact_point_notifiers.go [hunk 1]
```diff
@@ -1009,6 +1009,11 @@ func (n pagerDutyNotifier) schema() *schema.Resource {
 			Type: schema.TypeString,
 		},
 	}
+	r.Schema["url"] = &schema.Schema{
```
Comment (julienduchesne) [inline internal/resources/grafana/resource_alerting_contact_point_notifiers.go:1012]: It needs to be used in the `pack` and `unpack` functions

**Review by yuri-tceretian — CHANGES_REQUESTED** [review]
@gaurav1999 thanks for opening PR. Please add packing and unpacking. You can use commit https://github.com/grafana/terraform-provider-grafana/pull/1060/commits/3da42eb0bbd3d7ff5048cd82a7fb718e3eb94908 as example. 

To update documentation you only need to run `go generate` in the root of the repository.

**Comment by gaurav1999** [comment]
> @gaurav1999 thanks for opening PR. Please add packing and unpacking. You can use commit [3da42eb](https://github.com/grafana/terraform-provider-grafana/commit/3da42eb0bbd3d7ff5048cd82a7fb718e3eb94908) as example.
> 
> To update documentation you only need to run `go generate` in the root of the repository.

Thanks @yuri-tceretian , done

However if u see my previous commit, I indeed ran go generate in the root repo, it ended up removing summary from lots of doc files, i don't understand why then @julienduchesne added a commit to fix it, so I think for now the pointers are closed, let me know if anything else is needed, thanks a lot for guiding me around on this small steps!

**Comment by gaurav1999** [comment]
Wait, I ended up adding my work email here, let me see if I can update that in commits.

**Comment by gaurav1999** [comment]
> Wait, I ended up adding my work email here, let me see if I can update that in commits.

Done

**Review by julienduchesne — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**